pkg/apis/monitoring/v1alpha1/types.go

```diff
@@ -72,6 +72,10 @@ type MonitoringStackSpec struct {
 	// +optional
 	// +kubebuilder:default={replicas: 2}
 	PrometheusConfig *PrometheusConfig `json:"prometheusConfig,omitempty"`
+
+	// Define Alertmanager config
+	// +optional
+	AlertmanagerConfig *AlertmanagerConfig `json:"alertmanagerConfig,omitempty"`
```

Comment:
This avoids checking for nil values.

```suggestion
	AlertmanagerConfig AlertmanagerConfig `json:"alertmanagerConfig,omitempty"`
```